api/resources_portal/test/views/test_import_sra.py

```diff
@@ -0,0 +1,109 @@
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from resources_portal.models import Material
+from resources_portal.test.factories import GrantFactory, OrganizationFactory, UserFactory
+
+
+class ImportSRATestCase(APITestCase):
+    """
+    Tests importing via SRA.
+    """
+
+    def setUp(self):
+        self.test_accession = "SRR4199304"
```

Comment:
I think we want to use accessions that start with `SRP` rather than `SRR` because they correspond to studies rather than runs.

Reply:
Good call. I reworked the api calls to do this.